### PR TITLE
Spin bit:  Not implemented == disabled

### LINKED ISSUE #4666: Robert Wilton's Transport Discuss 1
Rob Wilton said:

> 1) I find it hard to understand why a server is allowed to independently decide
> whether or not to support the spin bit on a connection?  Shouldn't the client
> (or administrator of the client system) that opened the connection be able to
> choose whether they want the RTT to be monitorable via the spin bit?  What is
> the reasoning for allowing the server (or server administrator) to be able to
> independently be able to decide what is best for the client?

### PR DESCRIPTION
**Pull request by MikeBishop**
Fixes #4666.

### CODE REVIEW

File: draft-ietf-quic-transport.md [hunk 1]
```diff
@@ -5113,6 +5113,8 @@ discussed in {{?QUIC-MANAGEABILITY=I-D.ietf-quic-manageability}}.
 
 The spin bit is an OPTIONAL feature of this version of QUIC. A QUIC stack that
 chooses to support the spin bit MUST implement it as specified in this section.
+A QUIC stack that chooses not to support the spin bit MUST behave as if it were
```
Comment (ianswett) [inline draft-ietf-quic-transport.md:5116]: Do we need to add this sentence?

File: draft-ietf-quic-transport.md [hunk 2]
```diff
@@ -5126,7 +5128,9 @@ approximately one in eight network paths.
 When the spin bit is disabled, endpoints MAY set the spin bit to any value, and
 MUST ignore any incoming value. It is RECOMMENDED that endpoints set the spin
 bit to a random value either chosen independently for each packet or chosen
-independently for each connection ID.
+independently for each connection ID. One way to accomplish this is to adjust
```
Comment (ianswett) [inline draft-ietf-quic-transport.md:5131]: This seems like a useful addition.

**Review by ianswett — APPROVED** [review]

File: draft-ietf-quic-transport.md (hunk 1 above)
Comment (martinthomson) [reply draft-ietf-quic-transport.md:5116]: I have a simpler change:

> The spin bit is an OPTIONAL feature of this version of QUIC.   An endpoint that does not support this feature MUST disable it, as defined below.

File: draft-ietf-quic-transport.md (hunk 2 above)
Comment (martinthomson) [reply draft-ietf-quic-transport.md:5131]: No. No. No.  This does not work.  We discussed this.  It breaks the AEAD.

**Review by martinthomson — CHANGES_REQUESTED** [review]

File: draft-ietf-quic-transport.md (hunk 1 above)
Comment (janaiyengar) [reply draft-ietf-quic-transport.md:5116]: Use of "QUIC stack" is also odd. I like @martinthomson 's simpler suggestion.

**Review by janaiyengar — APPROVED** [review]
@martinthomson -- this is good now.

**Pull request merged**